letta/server/server.py

```diff
@@ -191,7 +190,14 @@ def db_error_handler():
     config.archival_storage_uri = settings.letta_pg_uri_no_default
 
     # create engine
-    engine = create_engine(settings.letta_pg_uri)
+    engine = create_engine(
```

Comment:
We might want to make these environment variables (maybe put them in settings.py) so they can be configured

Reply:
ack